cove_iati/settings.py

```diff
@@ -107,3 +110,18 @@
 
 # https://github.com/OpenDataServices/cove/issues/1098
 FILE_UPLOAD_PERMISSIONS = 0o644
+
+
+# Note on deploying new versions of this app the cache directory should be cleared
+# as new versions of requests & requests_cache libraries may use different versions
+# https://requests-cache.readthedocs.io/en/latest/user_guide/troubleshooting.html#potential-issues
+REQUESTS_SESSION_WITH_CACHING = CachedSession(
+    'iati_cove_cache',
+    backend=FileCache(
+        cache_name=env('REQUESTS_CACHE_DIR')
+    ),
+    expire_after=timedelta(days=1),  # Expire responses after one day
+    allowable_methods=['GET'],       # Cache POST requests to avoid sending the same data twice
```

Comment:
This comment is out of sync with what's set?

Reply:
Fixed